### DISCUSSION TITLE
pi5

### DISCUSSION THREAD

**Discussion by droidhackzor** [marcone/teslausb#886] [Q&A]
has anyone gotten this working on a pi5, i had this working on a pi zero w until it died but before that happened, i purchased a pi5 in the hopes that eventually i could figure out a way to use my spare 2TB nvme and m.2 for this since even a sata m.2 is faster than a micro-sd card, however, doing the same setup as for the zero, the pi5 doesn't seem to boot and ssh never connects, i am using a 256gig samsung high endurance and it just sits there. i have ordered a micro to full hdmi so i can use my test-tv to see whats going one during the setup but still not having any luck so figured i'd ask here while i wait for that hdmi. any help would be appreciated.

**Comment by marcone** [comment]
> doing the same setup as for the zero, the pi5 doesn't seem to boot

If by that you mean that you used the prebuilt TeslaUSB image, then that's expected. Pi 5 needs a more recent version of Raspberry Pi OS, so you will need to flash a standard Raspberry Pi OS image and use the "manual" TeslaUSB installation procedure described in the wiki.

**Comment by droidhackzor** [comment]
Many thanks, I'll look into that after work tomorrow

Sent from Samsung Galaxy smartphone.
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by jfryton** [reply]
Did you manage to get it working? Any observations to share?

**Comment by maxawad** [comment]
Is this working for Pi5?
